### PR TITLE
Checklist

### PR DESCRIPTION
**Pull request by hlaueriksson**
A proposal for a community plugin checklist :clipboard:

Please provide feedback 🙏

### CODE REVIEW

File: checklist.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Examples:
+
+✔️ `InputTyper-0.2.0-x64.zip`
+
+❌ `InputTyper.zip`
+
+The zip file should contain:
+
+- A folder with the same name as the plugin `AssemblyName`
```
Comment (hlaueriksson) [inline checklist.md:33]: Should this be the recommendation?

Comment (CoreyHayward) [reply checklist.md:33]: I think that the folder name should simply be matching the given name over the full assembly name. 
That seems to be the convention through the main PowerToys repo with both the Microsoft and community plugins such as `UnitConverter` that are built into PowerToys which you can find in the installation directory under the RunPlugins folder

Comment (hlaueriksson) [reply checklist.md:33]: Ah, yes! Now I found the official folders in `c:\Program Files\PowerToys\RunPlugins\`.

Comment (hlaueriksson) [reply checklist.md:33]: It's a bit strange though that the folder is called `Uri` in `c:\Program Files\PowerToys\RunPlugins\` and `Microsoft.Plugin.Uri` in `%LocalAppData%\Microsoft\PowerToys\PowerToys Run\Settings\Plugins`

Comment (hlaueriksson) [reply checklist.md]: @CoreyHayward I rewrote and reordered some of the text in the checklist. What do you think?

File: checklist.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+2. The last part of the assembly name:
+   - `Community.PowerToys.Run.Plugin.InputTyper`
+
+Community plugins are located here:
+
+- `%LocalAppData%\Microsoft\PowerToys\PowerToys Run\Plugins`
+
+For reference, the official plugins are located here:
+
+- `C:\Program Files\PowerToys\RunPlugins\`
```
Comment (CoreyHayward) [inline checklist.md:82]: This is only if you have installed PowerToys for all users. If installed for a specific user the default location is `%LocalAppData%\PowerToys\RunPlugins`

File: checklist.md (hunk 1 above)
Comment (CoreyHayward) [reply checklist.md]: Yeah this looks better, I agree it is a bit strange but I guess users are rarely interacting with the settings folder directly so probs readability of the installed plugins is a higher priority.

File: checklist.md (hunk 2 above)
Comment (hlaueriksson) [reply checklist.md]: Good catch! I pushed an update.

**Comment by CoreyHayward** [comment]
LGTM!

**Comment by hlaueriksson** [comment]
Hi!

@CoreyHayward
@davidegiacometti
@N0I0C0K
@Advaith3600
@Quriz
@KawaiiZapic
@dandn9
@bostrot
@zziger

You have contributed to this repo and helped to make it awesome. Thanks!

This PR is a proposal for a checklist on how to release community plugins, with examples of some do's and don'ts.
If you have time, you are welcome provide feedback 🙏

**Comment by Quriz** [comment]
LGTM! I think this will be really helpful!

**Comment by KawaiiZapic** [comment]
I just moved to `PluginStorage<T>` a month ago, now I have re-structed my release zip file and rename to `<name>-<version>-<platform>.zip`, and now it looks good to me.

**Comment by davidegiacometti** [comment]
LGTM! Nice doc! 😃

**Comment by Advaith3600** [comment]
Really well-put checklist!

**Pull request merged**